packages/flutter/test/material/range_slider_test.dart

```diff
@@ -1662,6 +1662,87 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('Range Slider thumb gets stroked when overlapping', (WidgetTester tester) async {
+    RangeValues values = const RangeValues(0.3, 0.7);
+
+    final ThemeData theme = ThemeData(
+      platform: TargetPlatform.android,
+      primarySwatch: Colors.blue,
+      sliderTheme: const SliderThemeData(
+        valueIndicatorColor: Color(0xff000001),
+        overlappingShapeStrokeColor: Color(0xff000002),
```

Comment:
What happens if overlappingShapeStrokeColor is null?

The SliderThemeData constructor's doc says:
```dart
  /// Create a [SliderThemeData] given a set of exact values. All the values
  /// must be specified.
```

None of the parameters are `@required` and all default to null (per our conventions for themes).

Reply:
It still passes because we expect to find a circle of theme.sliderTheme.overlappingShapeStrokeColor.

None of the parameters need to be specified. If it is not specified in the SliderTheme then the Slider has default parameters that are based of the ColorScheme